### ISSUE TITLE
Scripts assume reponame is git-playbook

### ISSUE THREAD

**Issue by aaronkorver** [thomasreinecke/git-playbook#1]
It looks like there is a hardcoding of the repo name "git-playbook" in the scripts.  I'm liking the way this project lays out things much better than other GH template engines.  The repo name should be passed in when generating the site or something configurable.

I'm not a Front-end dev, but I'll see if I have time to contrib back something that does this.

**Comment by anywhichway** [comment]
@aaronkorver I also thought stuff was hard-coded, but what I have found is some pathing issues. I have changed references in the production index.html from "/static/" to "./static" and changes references in manifest.json from "/static/<rest of path" to "./<rest of path>". I also changed the routes in pageConfig.json to match my project name. I have also found places in the service-worker.js and css files that need to be changed. Unfortunately, the changes have to be made after generation and I can't find the config file or generator code location to make a sitewide fix.
